ports/net/macathena-hostinfo/Portfile

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+PortSystem          	1.0
+PortGroup           	github 1.0
+
+github.setup        	mit-athena hostinfo 10.1.1
+
+name                    macathena-hostinfo
+version             	10.1.1
+categories          	net
+maintainers         	mit.edu:sipb-macathena
+platforms           	darwin
+description         	A utility to ask an Internet nameserver for host information
+long_description        The hostinfo utility fetches the Domain Name System records for the \
+ specified host name and prints them.
+checksums          rmd160  324deca6dbd49e7361879021cc0d1bf03f35862a \
+                    sha256  2241b5de60736afe913f9cdab34f9f53ed5d017c9a76c24e3008ff079922a8b2 \
+                    size    2870
+
+use_configure           no
+depends_lib             port:perl5
+
+patch {
+    reinplace "s|#!/usr/bin/perl|#!${prefix}/bin/perl|" ${worksrcpath}/hostinfo
```

Comment:
There's a perl5 portgroup that does much the same as the python portgroup; you should probably use that (though because hostinfo doesn't have a `Makefile.PL`, the equivalent of `setup.py`, you still will have to do some things manually. The path to perl is `${perl5.bin}`, and you should install the script into `${perl5.bindir}`, and call `perl5.setup macathena-hostinfo 10.1.1`. Though it might be faster to just use `${perl5.bin}` and manually set `depends_lib-append port:perl${perl5.default_branch}`, since you don't need any of the `Makefile.PL` handling.

Reply:
When I used the perl5 portgroup it tried to use the perl mirror sites to get the source and ignored the github.setup line. I tried putting things in a bunch of different orders and setting things individually and couldn't get it to work, does it source from the right site for you?

Reply:
It only tries to set things like master sites if you use `perl5.setup`, so you could do the latter thing I suggested. But I would have thought that it would be fine as long as `perl5.setup` comes before `github.setup`, though if you couldn't get that to work it's not worth continuing to fight with it. (I would still try using it for `${perl5.bin}` and `${perl5.default_branch}` without `perl5.setup`.)